android/src/main/java/com/aakashns/reactnativedialogs/modules/DialogAndroid.java

```diff
@@ -278,7 +309,10 @@ public void onClick(MaterialDialog materialDialog, DialogAction dialogAction) {
             mBuilder.showListener(new DialogInterface.OnShowListener() {
                 @Override
                 public void onShow(DialogInterface dialog) {
-                    callback.invoke("showListener");
+                    try {
+                        callback.invoke("showListener");
+                    } catch (Exception e) {
+                    }
```

Comment:
nit: You are missing the log message in the `showListener`. Thanks for logging exceptions in other places. Changes look good.